src/common/enums.js

```diff
@@ -202,3 +202,22 @@ export const systemEnvColors = {
   prd: 'green',
   qa: 'blue',
 };
+
+// Study creation status messages and icons
+export const statusyMessage = {
+  ERR: {
+    icon: 'warning sign',
+    text: 'This study may not have been created successfully',
+    class: 'text-red cursor-pointer noMargin',
+  },
+  SUC: {
+    icon: 'check circle',
+    text: 'This study has been created successfully',
+    class: 'text-green cursor-pointer noMargin',
+  },
+  STR: {
+    icon: 'clock',
+    text: 'The study creating is in progress',
```

Comment:
change up the wording to reflect that it's the external resources that are being created in the background and not the study: 

- **ERR** - `Error(s) creating external study resources. Click for details.`
- **SUC** - `External study resources created! Click for details.`
- **STR** - `External study resources creation in progress. Click for details.`